src/commands/ban.js

```diff
@@ -21,19 +21,33 @@ async function banHandler(interaction) {
 	const userIds = [...new Set(Array.from(users.matchAll(Discord.MessageMentions.USERS_PATTERN), match => match[1]))];
 
 	const bansListEmbed = new Discord.MessageEmbed();
-	bansListEmbed.setTitle('User Bans :thumbsdown:');
-	bansListEmbed.setColor(0xFFA500);
+	bansListEmbed.setTitle('User Bans');
+	bansListEmbed.setColor(0xa30000);
+	bansListEmbed.setThumbnail('attachment://mod-ban.png');
+
+	const image = new Discord.MessageAttachment('./src/images/mod/mod-ban.png');
 
 	for (const userId of userIds) {
 		const member = await interaction.guild.members.fetch(userId);
 		const user = member.user;
 
+		// Checks if they're above/equal to the executor
```

Comment:
This goes for all comments. We use the syntax from [Better Comments](https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments) now. I've been migrating our older code to use it, but newer code should use it by default

Reply:
Ah, I use VS2022, so I'm not certain if I'll be able to see them, but nonetheless, I'll try implementing that after I resolve everything else.